unstructured/embed/huggingface.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Optional
+
+import numpy as np
+from langchain.embeddings import HuggingFaceEmbeddings
+
+from unstructured.documents.elements import (
+    Element,
+)
+from unstructured.embed.interfaces import BaseEmbeddingEncoder
+from unstructured.ingest.error import EmbeddingEncoderConnectionError
+from unstructured.utils import requires_dependencies
+
+
+class HuggingFaceEmbeddingEncoder(BaseEmbeddingEncoder):
+    def __init__(
+        self,
+        model_name: Optional[str] = "sentence-transformers/all-MiniLM-L6-v2",
+        model_kwargs: Optional[dict] = {"device": "cpu"},
+        encode_kwargs: Optional[dict] = {"normalize_embeddings": False},
+        cache_folder: Optional[dict] = None,
+    ):
+        self.model_name = model_name
+        self.model_kwargs = model_kwargs
+        self.encode_kwargs = encode_kwargs
+        self.cache_folder = cache_folder
+
+        self.initialize()
+
+    @EmbeddingEncoderConnectionError.wrap
+    @requires_dependencies(
+        ["langchain", "huggingface"],
```

Comment:
Per the first code cell in https://python.langchain.com/docs/integrations/text_embedding/huggingfacehub
```suggestion
        ["langchain", "sentence_transformers"],
```
The list items here signify what might be lacking in terms of `pip install`s